humble-scraper/client
# Added basic implementation and styling of the item list with p

# Description

Added basic implementation and styling of the item list with placeholder items

## Link to work item
Closes #17 

## How to Test
Spin up the client app with yarn dev and check visual changes
![Untitled](https://user-images.githubusercontent.com/60013942/122141219-da304480-ce1a-11eb-9bc8-25684406c49c.png)
